### PR TITLE
C10 dk screen warranty registration 1

### PR DESCRIPTION
**Pull request by DennisKiselev**
Tested on c10 and rig

### CODE REVIEW

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py [hunk 1]
```diff
@@ -59,34 +59,35 @@
...
 				size_hint: (None,None)
-				width: dp(800)
-				height: dp(298)
+				width: dp(1.0*app.width)
+				height: dp(0.620833333333*app.height)
 				orientation: 'vertical'
 
 
 				Button:
+				    font_size: str(0.01875 * app.width) + 'sp'
```
Comment (LettyYeti) [inline src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:81]: check indentation

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py [hunk 2]
```diff
@@ -174,22 +175,23 @@
...
 				BoxLayout: 
 					size_hint: (None, None)
-					height: dp(122)
-					width: dp(244.5)
-					padding: [0, 0, 184.5, 0]
+					height: dp(0.254166666667*app.height)
+					width: dp(0.305625*app.width)
+					padding:[0, 0, dp(0.230625)*app.width, 0]
 					Button:
+					    font_size: str(0.01875 * app.width) + 'sp'
```
Comment (LettyYeti) [inline src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:190]: check indentation

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py [hunk 3]
```diff
@@ -208,76 +210,81 @@
...
 					size_hint: (None, None)
-					height: dp(122)
-					width: dp(244.5)
-					padding: [dp(192.5), 0, 0, 0]
+					height: dp(0.254166666667*app.height)
+					width: dp(0.305625*app.width)
+					padding:[dp(0.240625)*app.width, 0, 0, 0]
 
 					Button:
+					    font_size: str(0.01875 * app.width) + 'sp'
```
Comment (LettyYeti) [inline src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:239]: check indentation

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py [hunk 4]
```diff
@@ -208,76 +210,81 @@
...
+    def update_contact_us_font_sizes(self): # Update both labels together to make it look nicer
+        if self.l.get_text_length(self.contact_us_at_support.text) > 70:
+            self.cant_use_web_label.font_size = 0.02125 * Window.width
+            self.contact_us_at_support.font_size = 0.02125 * Window.width
+        else:
+            self.cant_use_web_label.font_size = 0.025 * Window.width
+            self.contact_us_at_support.font_size = 0.025 * Window.width
+
+    def go_to_factory_settings(self):
+        popup_warranty.PopupFactorySettingsPassword(self.start_seq.am)
```
Comment (LettyYeti) [inline src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:287]: make a note to come back to popups

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py [hunk 5]
```diff
@@ -208,76 +210,81 @@
...
+
+    def quit_to_console(self):
+        popup_warranty.QuitToConsoleWarranty(self.start_seq.sm)
```
Comment (LettyYeti) [inline src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:290]: make a note to come back to popups

**Review by LettyYeti — COMMENTED** [review]
See comments

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py (hunk 1 above)
Comment (DennisKiselev) [reply src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:81]: This is just github being weird and displaying tabs/spaces differently, it looks correct in the editor and works properly

Comment (DennisKiselev) [reply src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:81]: image

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py (hunk 2 above)
Comment (DennisKiselev) [reply src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:190]: Same as above

File: src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py (hunk 3 above)
Comment (DennisKiselev) [reply src/asmcnc/apps/start_up_sequence/warranty_app/screens/screen_warranty_registration_1.py:239]: Same as above

**Review by LettyYeti — APPROVED** [review]

**Pull request merged**